optaplanner-core/src/test/resources/org/optaplanner/core/config/solver/testSolverConfigWithoutNamespace.xml

```diff
@@ -156,4 +156,12 @@
       </acceptor>
     </localSearch>
   </partitionedSearch>
+  <loop>
+    <constructionHeuristic />
+    <localSearch />
+    <customPhase />
+    <exhaustiveSearch />
+    <noChangePhase/>
+    <partitionedSearch />
+  </loop>
```

Comment:
@ge0ffrey This is what the solver config looks like. `<loop>` is a phase like any other - can go anywhere where other phases can go, can have terminations, can be part of other phases.

Reply:
How does the termination of a loop phase compose with terminations configured on individual phases inside the loop?

Reply:
It doesn't. The nested phase needs to terminate according to its own termination, at which point the loop phase termination's evaluated. This is likely a place where we want some real-world examples of things to try and test.